code/modules/power/apc/apc.dm

```diff
@@ -495,11 +495,10 @@
 	if(issilicon(user))
 		var/mob/living/silicon/ai/AI = user
 		var/mob/living/silicon/robot/robot = user
-		if(aidisabled || malfhack && istype(malfai))
-			if((istype(AI) && (malfai!=AI && malfai != AI.parent)) || (istype(robot) && (robot in malfai.connected_robots)))
-				if(!loud)
-					to_chat(user, "<span class='danger'>\The [src] has AI control disabled!</span>")
-				return FALSE
+		if(aidisabled || malfhack && istype(malfai) && ((istype(AI) && (malfai != AI && malfai != AI.parent)) || (istype(robot) && (robot in malfai.connected_robots))))
```

Comment:
```suggestion
		if(aidisabled || (malfhack && istype(malfai) && ((istype(AI) && (malfai != AI && malfai != AI.parent))) || (istype(robot) && (robot in malfai.connected_robots))))
```
This would help with readability, but not a requirement for merge